jetty-ee10/jetty-ee10-servlet/src/main/java/org/eclipse/jetty/ee10/servlet/ServletChannel.java

```diff
@@ -649,12 +644,10 @@ else if (ExceptionUtil.areNotAssociated(cause, x))
 
         if (LOG.isDebugEnabled())
             LOG.debug("!handle {} {}", action, this);
-
-        boolean suspended = action == Action.WAIT;
-        return !suspended;
     }
 
     /**
+     * TODO Remove if really not needed
```

Comment:
This method is called by `handle()`, so this TODO seems outdated.

But this method can be made `private`.